packages/features/audit-logs/constants.ts

```diff
@@ -0,0 +1,79 @@
+import { AuditLogBookingTriggerEvents, AuditLogTriggerTargets } from "@calcom/prisma/enums";
+
+import type { DefaultAppSettingOptionEntry } from "./types";
+import { DefaultAppSettingsOptions } from "./types";
+import { getHref } from "./utils";
+
+export const availableTriggerEvents: { [key: string]: string | any } = {
```

Comment:
availableTriggerEvents and availableTriggerTargets are used to render the Event Toggles